index.html

```diff
@@ -29,8 +29,8 @@ <h3 id="job">student web developer</h3>
       </ul>
     </nav>
     <figure>
-      <img src="images/daniel-eicher.png" alt="" width="500" height="500">
-      <figcaption>welcome to my personal web page</figcaption>
+      <img src="images/daniel-eicher.png" alt="" width="700" height="500">
```

Comment:
Why do you leave the alt tag empty?

Reply:
oh, i forgot to fill-in information when i change the random picture - thx a lot